Posts/meeting-23mar- gem updates

```diff
@@ -0,0 +1,376 @@
+---
+title: Raw IRC chat Mar 23rd 2016
+author: avsm (Anil Madhavapeddy)
+abstract: Raw IRC meeting notes on #mirage on Mar23 2016
+---
+
+```
+### Roll Call  
```

Comment:
Something we have in mirage-www that seemed to work quite well was having a
record per author that contains thing like name, URL, etc. (searching by
author or other tag fields also useful -- so there are fields like author,
tag, etc that are lists of values.)

On 24 March 2016 at 15:15, Enguerrand Decorne <notifications@github.com>
wrote:

> In Posts/meeting-23mar- gem updates
> <https://github.com/hannesm/canopy-data/pull/6#discussion_r57332483>:
>
> > @@ -0,0 +1,376 @@
> > +---
> > +title: Raw IRC chat Mar 23rd 2016
> > +author: avsm (Anil Madhavapeddy)
> > +abstract: Raw IRC meeting notes on #mirage on Mar23 2016
> > +---
> > +
> > +```
> > +### Roll Call
>
> Authors at the moment are just a plain string, you can put pretty much
> anything in there. Would it be useful to be able to find articles using the
> author's name ? i can change that if needed
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/hannesm/canopy-data/pull/6/files/823892655a831ec7aa4f35cfc7cc35498bcdfee6#r57332483>
>



-- 
Richard Mortier
mort@cantab.net